src/serverOperations/booksServerOperations.js

```diff
@@ -0,0 +1,75 @@
+import { bases } from '../enums';
+import {
+    createRecord,
+    deleteRecord,
+    getRecords,
+    updateRecord,
+} from './crudOperations';
+import { fetchFromBase } from './fetchFromBase';
+
+export function getBookServerOperations(sessionID) {
+    return {
+        createBook: async (title, author, amount) => {
+            return await createRecord(
+                bases.BOOKS,
+                { title, author, amount },
+                sessionID,
+            );
+        },
+
+        /**
+         * @param params {{ title?:string, author?:string, amount?:number }}
```

Comment:
So here are a few ways how you should do this:
- you may keep working with JSDoc, but you will need to add them everywhere, no exceptions
- if you really want strong typing you may switch to TypeScript - at this moment you still don't have a big codebase, so it is possible
- or you may just get rid of JSDocs and work in plain JS.

Decision is up to you, I suggest options 2-3, but please do not mix these approaches. It is even more confusing now, because you add docs to `GET` requests which usually are quite descriptive from their method/function/endpoint name